### ISSUE TITLE
Bowling FinalShot #2

### ISSUE THREAD

**Issue by adavila17** [Aubron/scoreshots-templates#74]
![ss_bowling_finalshot2](https://user-images.githubusercontent.com/26416133/30543009-7437adae-9c4f-11e7-9471-fcd793eb4c72.jpg)

**Comment by adavila17** [comment]
Here's what a 7 baker would look like.

Hopefully, the angle of the design isn't an issue.

![ss_bowling_finalshot2_7baker](https://user-images.githubusercontent.com/26416133/30558174-8b033e0c-9c7e-11e7-8214-671ef772435e.jpg)

**Comment by ValentineRain** [comment]
Looks fancy! I always thought bowling scoring was kind of confusing.

**Comment by adavila17** [comment]
Try team bowling then you'll love it!

**Comment by Aubron** [comment]
Add sponsor logos.

**Comment by adavila17** [comment]
New Template ID: 677
New Category: Bowling

**Needs Dynamism**
Drop Down labeled "Number of Bakers"
1-7 Bakers
Hide elements if lower than 7 Bakers.
Slightly increase the size of the middle area when less than 7 Bakers as well.

**Comment by kelvin-im** [comment]
Working on this now.

**Comment by kelvin-im** [comment]
This template should be dynamic now. May need to test to make sure objects are properly scaled. Ready for review/testing @adavila17, @ValentineRain.

PR: https://github.com/Aubron/scoreshots/pull/304

**Comment by ValentineRain** [comment]
I have reached out to Arkansas State, as well as Lawrence Tech and Blue Mountain College, for feedback on the designs. In order to keep things moving forward I'm going to go ahead and review these templates, and we can reopen the task for edits later if need be.

**Comment by ValentineRain** [comment]
We'll need to take a background step out of this one after merging. 

Are the sponsor logo spaces supposed to be sideways? It's kinda goofy.

![rotated logos](https://user-images.githubusercontent.com/26412274/31026605-1a07cf52-a515-11e7-8458-a9bdf5cf1241.png)

Finally, can you please change one of the logo backgrounds slightly; dynamic templates cant have colors individually changed, so since they're collapsed they must be the same color. As a result, it is currently impossible to match your preview image.

**Comment by adavila17** [comment]
try again

**Comment by ValentineRain** [comment]
All those things are fixed, minus the background space which we'll pick up after merging.

**Comment by ValentineRain** [comment]
I'm going to paste our most recent feedback for all of he bowling templates in the relevant threads. We should talk about how to incorporate this stuff in the next review meeting, because it is waaaay over my head.

This is from Arkansas State:

> As for feedback, I think they look great. My only concern would be that you can get seven games on the first one in there for seven-game Baker matches, but that does look great and can be used for the five-game Baker matches. I think it would be good to add somewhere on there the total score. And I’m getting real technical and bowling is about as weird a sport as there is, but the five-game Baker matches take the total pins to determine the winner so maybe something that adds them up beside the team names.
>  
> The second one works great for the seven-game Baker matches that are usually on the final day of the tournaments. It would also be cool to have enough space to type in say 1,015 for team one to 1,002 for team two because that is what the scores are on five-game Baker matches and traditional team matches. But I like that one a lot, looks clean and right to the point.
>  
> Another design that would be cool, and again, I know I’m being hard to follow because bowling is so different, but on the second day of tournaments they play traditional team matches so five individual players bowl 10 frames each. It would be neat to have a graphic that lists all five players in the lineup and able to put their score beside their name with the total team score at the bottom and just a place to put the opponent’s total score. We don’t always get the scores for the individuals, but when we do that would be a cool graphic to add.
>  
> That’s a good start though and they all look great to me! Bowling is just really complicated. It took me a lot of time to get it as well, so I’m good to answer any other questions because I know a lot of that is confusing.
>  
> Thanks!

**Comment by adavila17** [comment]
So one new design with players and totals and some additions to the finalshots.

**Comment by ValentineRain** [comment]
This came in from LTU:

> I would use them for sure, however in the naia we only play in tournaments. I would love one that has a top five for teams and for players.

**Comment by Aubron** [comment]
@adavila17 to take on feedback because he actually understands it.

**Comment by adavila17** [comment]
added total pins to template

**Comment by ValentineRain** [comment]
Reopening because it still isn't visible; it needs a preview image. Sorry!

**Comment by adavila17** [comment]
added
